include/irr/asset/filters/CSummedAreaTableImageFilter.h

```diff
@@ -319,10 +326,10 @@ class CSummedAreaTableImageFilter : public CMatchedSizeInOutImageFilterCommon, p
 							{
 								uint8_t offset = &itrValue - current;
 
-								if (maxDecodeValues[offset] > itrValue)
+								if (maxDecodeValues[offset] < itrValue)
 									maxDecodeValues[offset] = itrValue;
 
-								if (minDecodeValues[offset] < itrValue)
+								if (minDecodeValues[offset] > itrValue)
 									minDecodeValues[offset] = itrValue;
```

Comment:
what happened here?

Reply:
It was wrong, I have seen it recently while using the lambdas with those buffers